### PR TITLE
HIVE-26468: Replace FlatBuffers artefact from com.github.davidmoten w…

### PR DESCRIPTION
**Pull request by slachiewicz**
…ith artefact from Google

### CODE REVIEW

File: serde/pom.xml [hunk 1]
```diff
@@ -81,7 +81,7 @@
       <version>${hppc.version}</version>
     </dependency>
     <dependency>
-      <groupId>com.github.davidmoten</groupId>
+      <groupId>com.google.flatbuffers</groupId>
```
Comment (ayushtkn) [inline serde/pom.xml:84]: does this change introduce any incompatibilities?

Comment (slachiewicz) [reply serde/pom.xml:84]: I don't think so (see warning message in jira ticket), previously for this module we have 2 dependencies from different groups and shade plugin may select one of them randomly. My change makes clear to use only official artefacts from one source.

Comment (slachiewicz) [reply serde/pom.xml:84]: @ayushtkn anything more I can help with review?

Comment (ayushtkn) [reply serde/pom.xml:84]: Should be fine then. Sorry I forgot about this.

**Review by ayushtkn — APPROVED** [review]
LGTM

**Pull request merged**